src/containerapp/azext_containerapp/_constants.py

```diff
@@ -83,8 +83,8 @@
     timeout: 1800
 """
 
-ACA_BUILDER_BULLSEYE_IMAGE = "mcr.microsoft.com/oryx/builder:debian-bullseye-20231025.1"
-ACA_BUILDER_BOOKWORM_IMAGE = "mcr.microsoft.com/oryx/builder:debian-bookworm-20231025.1"
+ACA_BUILDER_BULLSEYE_IMAGE = "mcr.microsoft.com/oryx/builder:debian-bullseye-20231107.2"
+ACA_BUILDER_BOOKWORM_IMAGE = "mcr.microsoft.com/oryx/builder:debian-bookworm-20231107.2"
```

Comment:
@cormacpayne Could you please confirm that this change looks OK to you?

Reply:
These are the latest base builders that we've published so it looks OK to me

Reply:
Thanks Cormac!